### PR TITLE
First commit, added task list

### PR DESCRIPTION
**Pull request by Zoltekk**
First commit, many more to come

### CODE REVIEW

File: task_list [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Napisz test automatyczny, który zaloguje się na stronie https://www.saucedemo.com i doda 3 randomowe rzeczy do koszyka.
+Test powinien sprawdzić czy kwota podliczona przy checkoucie jest prawidłowa
```
Comment (Glovacky) [inline task_list]: Please translate it to english and change the file extension to .txt

**Review by Glovacky — CHANGES_REQUESTED** [review]
Please fix the stuff I commented

File: task_list (hunk 1 above)
Comment (Zoltekk) [reply task_list]: Fixed

**Review by Glovacky — APPROVED** [review]
Everything OK

**Pull request merged**